tests/Oryx.Integration.Tests/Node/NodeTestWithAppInsightsConfigured.cs

```diff
@@ -22,33 +22,26 @@ public NodeTestWithAppInsightsConfigured(ITestOutputHelper output, TestTempDirTe
         }
 
         [Theory]
-        [MemberData(nameof(TestValueGenerator.GetNodeVersions_SupportDebugging),
+        [MemberData(nameof(TestValueGenerator.GetNodeVersions),
             MemberType = typeof(TestValueGenerator))]
         public async Task CanBuildAndRun_NodeApp_WithAppInsights_Configured(string nodeVersion)
         {
             // Arrange
             var appName = "linxnodeexpress";
             var volume = CreateAppVolume(appName);
             var appDir = volume.ContainerDir;
-            var spcifyNodeVersionCommand = "--platform nodejs --language-version=" + nodeVersion;
+            var spcifyNodeVersionCommand = "--platform nodejs --platform-version=" + nodeVersion;
             var aIKey = "APPINSIGHTS_INSTRUMENTATIONKEY";
+            var aIEnabled = "APPLICATIONINSIGHTS_CODELESS_ENABLED";
```

Comment:
This will likely be renamed. Putting a note here for when I know for sure what it will be instead.

Reply:
Can you rename this to `ApplicationInsightsAgent_EXTENSION_ENABLED`?

Reply:
I will, but waiting for the confirmation about the casing .. new variable's casing looks bit unorthodox :)

Reply:
It can be made all uppercase